### DISCUSSION TITLE
Classification by Voting Power

### DISCUSSION THREAD

**Discussion by dbwebapp** [BitCannaGlobal/WeedWallet#95] [General]
When User want to order by [Voting Power], one line should be on first position. See below : 
![Screenshot 2023-05-26 at 20 29 20](https://github.com/BitCannaGlobal/WeedWallet/assets/96944913/60d86a89-79e6-4cab-8ec6-7a5ec740a4e5)

**Comment by TJ-BitCanna** [comment]
Good find :)

**Comment by TJ-BitCanna** [comment]
Suggested issue:
- fix sorting of validators on voting power

**Comment by TJ-BitCanna** [comment]
Tested and approved.
